ur_description/urdf/ur10_joint_limited_robot.urdf

```diff
@@ -282,6 +282,22 @@
       <mechanicalReduction>1</mechanicalReduction>
     </actuator>
   </transmission>
+  <link name="base"/>
+  <joint name="base_link-base" type="fixed">
+    <!-- NOTE: this rotation is only needed as long as base_link itself is
+                 not corrected wrt the real robot (ie: rotated over 180
+                 degrees)
+      -->
```

Comment:
do we need this comment?
I think not as the ```base_link``` will not be corrected due to backwards compatibility...